docs/resources/aws_elasticloadbalancingv2_listeners.md

```diff
@@ -0,0 +1,60 @@
+---
+title: About the aws_elasticloadbalancingv2_listeners Resource
+platform: aws
+---
+
+# aws\_elasticloadbalancingv2\_listeners
+
+Use the `aws_elasticloadbalancingv2_listeners` InSpec audit resource to test the properties of a collection of AWS ElasticLoadBalancingV2 Listeners.
+
+## Syntax
+
+An `aws_elasticloadbalancingv2_listeners` resource block returns all ElasticLoadBalancingV2 Listeners.
+    describe aws_elasticloadbalancingv2_listener(listener_arn: "test_arn") do
+      its { should exist }
+    end
+    
+#### Parameters
+
+##### listener_arn _(required)_
```

Comment:
Is this required? Or should it be plural `listener_arns`?

Reply:
Updated, Now it accepts ARN of a Load Balancer
Thanks